schemas/registration.yaml

```diff
@@ -0,0 +1,35 @@
+registration:
+  description: Registration initialization
+  type: object
+  x-examples:
+    example-1:
+      flow: 231125d7-11d5-4e0f-80e9-5081003ae8ca
+      properties:
+        id:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
```

Comment:
`properties` seems to be misindented, and the example ("flow") still doesn't match the schema ("id")

Reply:
It also does not match the reality. I will update it and align it with the real apis responses, but decided to extend the scope of this PR not even further. Only clear and obvious mistakes were fixed.